spec/senior_teacher_spec.rb

```diff
@@ -18,9 +18,9 @@
   end
 
   it "has a performance rating" do
-    expect{jan.performance_rating}.not_to raise_error
-    expect(jan).to receive(:performance_rating)
-    jan.performance_rating
+    expect{jan.perform_rating}.not_to raise_error
+    expect(jan).to receive(:perform_rating)
```

Comment:
The tests should not be changed at all.  If you needed to change the test, then something was incorrect with your implementation.  Refactoring is modify without breaking existing functionality (tests).